docs/topics/proc_generating-discovery-manifest.adoc

```diff
@@ -0,0 +1,42 @@
+:_newdoc-version: 2.15.0
+:_template-generated: 2024-2-21
+:_mod-docs-content-type: PROCEDURE
+
+[id="generating-discovery-manifests_{context}"]
+= Generating discovery manifests
+
+[role="_abstract"]
+To generate discovery manifest, you must retrieve the application configuration from the Cloud Foundry (CF) source platform instance. {ProductShortName} generates a discovery manifest that has the application configuration, which you can later use to generate assets (deployment manifests).
+
+You can retrieve configuration for one or more applications at a time.
+
+.Prerequisites
+//should the users be logged in to the CF instance to generate discovery manifest?
+* You are logged in to the Cloud Foundry instance.
+
+* You configured a source platform for the applications for which you need to retrieve configurations.
+
+.Procedure
+
+. In the {ProductShortName} user interface, select the *Migration* working mode.
+
+. Click *Application Inventory* in the left menu bar.
```

Comment:
Please remove and start the nect step with ~ "On the Application Inventory page..."